NOTICE

```diff
@@ -0,0 +1,15 @@
+Copyright 2020 RDK Management
+Licensed under the Apache License, Version 2.0
+
+Copyright 2014 Fraunhofer FOKUS
+Licensed under the Apache License, Version 2.0
+
+Copyright 2015 Linaro Ltd
+Licensed under the Apache License, Version 2.0
+
+Copyright (C) 2016 TATA ELXSI
+Copyright (C) 2016 Metrological
```

Comment:
Please organise a little differently:
Under line 11, add "Licensed under the BSD-2 license" and a blank line, so it is clear that the two authors are credited as a pair.

Reply:
NOTICE file updated with review comment and submitted the change.